eth-contracts/migrations/8_delegate_manager_migration.js

```diff
@@ -74,8 +86,11 @@ module.exports = (deployer, network, accounts) => {
       claimsManagerProxyKey,
       _lib.toBN(0),
       'setDelegateManagerAddress(address)',
-      _lib.abiEncode(['address'], [claimsManagerAddress]),
+      _lib.abiEncode(['address'], [delegateManagerProxy.address]),
```

Comment:
@hareeshnagaraj i think this was a bug, could you double check it wasn't intentional?

Reply:
yes i think it was nice catch